app/uk/gov/hmrc/gform/models/DateExpr.scala

```diff
@@ -31,6 +31,7 @@ object DateExpr {
     case ExactDateValue(year, month, day) => DateExpr(year, month, day)
     case n @ NextDateValue(_, _)          => nextDateExpr(n)
     case p @ PreviousDateValue(_, _)      => previousDateExpr(p)
+
```

Comment:
We need to fix this compiler warning:
```
[warn]  [E1] app/uk/gov/hmrc/gform/models/DateExpr.scala:29:48:
[warn]       match may not be exhaustive.
[warn]       It would fail on the following inputs: FirstDayValue(_, _), LastDayValue(_, _)
[warn]         def fromDateValue(dv: DateValue): DateExpr = dv match {
```